tests/expected/panic-hook/main.rs

```diff
@@ -0,0 +1,11 @@
+use std::panic;
+
+fn main() {
+    panic::set_hook(Box::new(|_| {
+        println!("Custom panic hook");
```

Comment:
Should we be checking for this?

Reply:
I tried to, but it didn't work. I'll file an issue and link it.

Reply:
Are you closing #208?

Reply:
Yes, this PR will close it.

Reply:
Makes sense to me.